rollout/trafficrouting/istio/controller.go

```diff
@@ -174,32 +174,67 @@ func (c *IstioController) EnqueueRolloutFromIstioVirtualService(vsvc interface{}
 	}
 }
 
+func (c *IstioController) validateRolloutVirtualServicesConfig(r *v1alpha1.Rollout) error {
```

Comment:
This one should be called at https://github.com/argoproj/argo-rollouts/pull/1381/files#diff-ee634f180c1529c412a4207985224a3d0670f7332d00e967fa72f090af273f28L216 as well

Reply:
[copy pasting the same response from previous review comment.]

@huikang 
Please check the below code. getRolloutReferencedResources  is called before ValidateRolloutReferencedResources.
As i mentioned, the reason for introducing this validation in GetReferencedVirtualServices is because this method iterates the virtualservices to fill the k8s virtualservice record references. So, it doesn't really make sense to introduce this validation in ValidateVirtualService. But i can do one thing if required, before calling the method GetReferencedVirtualServices i can validate the multi VSes invalid config.
[https://github.com/argoproj/argo-rollouts/blob/2620490c9ab96e9da43b9c014fdf802c5e87dfa3/rollout/controller.go#L468](url)

Please let  me know your thoughts